gradle/release.gradle

```diff
@@ -11,10 +11,9 @@ buildscript {
     }
 }
 
-ext.repo = Grgit.open(rootDir)
+apply plugin: org.ajoberstar.gradle.git.base.GrgitPlugin
 
 release {
-    grgit = Grgit.open(rootDir)
     versionStrategy Strategies.FINAL
     defaultVersionStrategy = Strategies.FINAL
```

Comment:
I believe previously this was Strategies.SNAPSHOT and I changed it to FINAL? Is that still Ok?